lib/qless/server/views/layout.erb

```diff
@@ -91,7 +91,7 @@
      * flash a message to that effect on the page */
     var move = function(jid, queue, cb) {
       _ajax({
-        url: '<%= u "/move" %>',
+        url: <%== (u "/move").to_json %>,
```

Comment:
that's a weird way to call a method 😄 
`u("/move").to_json` vs `(u "/move").to_json` I guess neither of them are very readable...

Reply:
I think `u("/mode").to_json` is better; I'll change it to that.